agent/reconciler/host_reconciler.go

```diff
@@ -118,10 +122,16 @@ func (r *HostReconciler) reconcileNormal(ctx context.Context, byoHost *infrastru
 			logger.Info("Skipping installation of k8s components")
 		} else if r.UseInstallerController {
 			if byoHost.Spec.InstallationSecret == nil {
```

Comment:
Sorry, I'm confused here. Why we have removed the `!conditions.IsTrue(byoHost, infrastructurev1beta1.K8sComponentsInstallationSucceeded)` check?

Reply:
When I added this condition, it was not moving into the code block. Not sure how conditions.IsTrue() ?